src/Types.ts

```diff
@@ -153,18 +153,19 @@ export type SwapAmountJSON = {
   paramTypes: SolidityFunctionParamType[]
 }
 
-export type SegmentFunctionName = 
-  'useBit' |
-  'marketSwapExactInput' |
-  'requireBitUsed' |
-  'requireBitNotUsed' |
-  'requireBlockNotMined' |
-  'requireBlockMined' |
-  'requireUint256LowerBound' |
-  'requireUint256UpperBound' |
-  'limitSwapExactInput' |
-  'blockInterval' |
-  'swap01'
+export enum SegmentFunctionName {
```

Comment:
why do we need this?

If we are going to change these to enums, we should do it consistently for all types that use this pattern

Reply:
it's actually a breaking change. so we might need to fix values in sdk or monorepo code using the types we convert from union to enums. e.g. here: https://github.com/brinktrade/brink-sdk/pull/142/files#diff-01db4c05730ffa0bab3256607971340bf3a63c53c7b8da7a4f807d756b195bbeR50

so i suggest we don't update every union types in order to avoid breaking changes. because we moves some to enums only because, we someitmes need to fetch the possible values which we do not need to do for every of them.